### PR TITLE
Chore: ApprovalCommentForm CSS fixes

### PR DESCRIPTION
**Pull request by pramodsum**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @pramodsum has signed the CLA. Thanks for the pull request!

File: src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.scss [hunk 1]
```diff
@@ -38,6 +38,10 @@
                 margin-right: 0;
                 margin-top: 0;
             }
+
+            .btn:last-child {
```
Comment (pramodsum) [inline src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.scss]: Adds 10px padding between ACF Post/Cancel buttons

File: src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.scss [hunk 2]
```diff
@@ -106,10 +110,6 @@
         .bcs-comment-add-approver-date-input {
             margin-left: 0;
             margin-right: 0;
-
-            .date-picker-open-btn {
```
Comment (pramodsum) [inline src/components/ContentSidebar/ActivityFeed/approval-comment-form/ApprovalCommentForm.scss]: Vertically centers calendar icon

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #438

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

**Pull request merged**